### PR TITLE
Manual session creating

### LINKED ISSUE #105: [Feature Request] Add a way to send the terminate signal to container from UI.
CUrrently when I run my specs from local machine against GGR, if I want to stop test execution, I should wait until current tests are finished, even if I stopped them locally. 
It would be nice to have some button to click to terminate test execution and stop container.
Also it would be cool to secure the element with password or permissions.

### LINKED ISSUE #114: [Feature request] Possibility to start new sessions for manual testing
It would be quite convenient to have a possibility of starting a new browser session directly in selenoid-ui (and then work with a browser via VNC session directly in selenoid-ui). 
This could be used for manual tests / troubleshooting issues with automated tests in specific browser versions (replicate Saucelabs "manual testing" feature)

Preconditions:
- Browser images with VNC should be used.
- Figure out what to do if `selenoid-ui` is used with `ggr-ui / ggr`, as creating a new session requires username/password.

### PR DESCRIPTION
**Pull request by fote**
Fixes #114
Fixes #105

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -10,3 +10,5 @@ github.com/koding/websocketproxy v0.0.0-20180716164433-0fa3f994f6e7 h1:UPc4az2SL
 github.com/koding/websocketproxy v0.0.0-20180716164433-0fa3f994f6e7/go.mod h1:Nn5wlyECw3iJrzi0AhIWg+AJUb4PlRQVW4/3XHH1LZA=
 github.com/rakyll/statik v0.1.5 h1:Ly2UjURzxnsSYS0zI50fZ+srA+Fu7EbpV5hglvJvJG0=
 github.com/rakyll/statik v0.1.5/go.mod h1:OEi9wJV/fMUAGx1eNjq75DKDsJVuEv1U0oYdX6GX8Zs=
+github.com/rakyll/statik v0.1.6 h1:uICcfUXpgqtw2VopbIncslhAmE5hwc4g20TEyEENBNs=
```
Comment (vania-pooh) [inline go.sum]: This needs `go mod tidy`.

File: go.mod [hunk 2]
```diff
@@ -5,5 +5,5 @@ require (
 	github.com/aerokube/util v0.0.0-20181001032247-3a9b3f70da09
 	github.com/gorilla/websocket v1.4.0 // indirect
 	github.com/koding/websocketproxy v0.0.0-20180716164433-0fa3f994f6e7
-	github.com/rakyll/statik v0.1.5
+	github.com/rakyll/statik v0.1.6
```
Comment (lanwen) [inline go.mod]: why this change?

File: ui/package.json [hunk 3]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.1.3",
```
Comment (lanwen) [inline ui/package.json]: please avoid any ui library here, its not that hard to define own components for the needs of the feature

File: ui/src/components/Sessions/index.js [hunk 4]
```diff
@@ -1,14 +1,42 @@
...
 const Sessions = (props) => {
     const {sessions = {}} = props;
     const list = Object.keys(sessions);
 
+    const [loading, setLoading] = useState([])
+
+    const deleteSession = (e, id) => {
+      e.preventDefault()
+      e.stopPropagation()
+      setLoading([...loading, id])
```
Comment (lanwen) [inline ui/src/components/Sessions/index.js]: please don't use state, pass the callback via rx-connect - you can see a great tutorial on that 
https://bsideup.gitbooks.io/rxconnect/content/

so the components became stateless - you can spot an example in the Session/index.js component

File: ui/src/components/Sessions/index.js [hunk 5]
```diff
@@ -1,14 +1,42 @@
...
+
+      const deleteSession = Observable.ajax({
+        url: `/wd/hub/session/${id}`,
+        method: 'DELETE',
+      })
+
+      deleteSession.subscribe(
+        res => {
+          console.log(res)
```
Comment (lanwen) [inline ui/src/components/Sessions/index.js]: please don't log anything in console until its really the only way to give a feedback. Here is not that case

File: ui/package.json (hunk 3 above)
Comment (lanwen) [reply ui/package.json]: the simplest example of the component is `components/Status`

**Comment by lanwen** [comment]
Great! Can you share a screenshot of how it looks like for you?

**Comment by lanwen** [comment]
also  #105

**Comment by fote** [comment]
Снимок экрана 2019-07-05 в 15 12 58
Снимок экрана 2019-07-05 в 15 15 24

**Comment by lanwen** [comment]
Thank you for the contribution, closed in favour of #155
